src/lib/input/input.spec.ts

```diff
@@ -438,6 +438,26 @@ describe('MatInput without forms', () => {
     expect(inputEl.disabled).toBe(true);
   }));
 
+  it('supports the disabled attribute as binding for select', fakeAsync(() => {
+    const fixture = createComponent(MatInputSelect);
+    fixture.detectChanges();
+
+    const formFieldEl =
+      fixture.debugElement.query(By.css('.mat-form-field')).nativeElement;
```

Comment:
Just noticed this, but line overflow indent should be +4 rather than +2
(here and elsewhere)